backend/src/sms/services/sms-callback.service.ts

```diff
@@ -21,9 +22,25 @@ const isAuthenticated = (
 
   const credentials = Buffer.from(secret, 'base64').toString('utf8')
   const [username, password] = credentials.split(':')
-  const plainTextPassword =
-    username + messageId + campaignId + config.get('smsCallback.callbackSecret')
-  return bcrypt.compareSync(plainTextPassword, password)
+  const plainTextPassword = username + messageId + campaignId
```

Comment:
Could be useful to put this format into `shared` for easier modification in future.

Reply:
This is only applicable for sms (duplicated once only) so I prefer keeping it here